frontend/src/components/Footer/HallOfFame.tsx

```diff
@@ -0,0 +1,94 @@
+import { Dialog, DialogTitle, Typography, IconButton, DialogContent, makeStyles, Grid, Box } from "@material-ui/core";
+import { Close } from "@material-ui/icons";
+
+const rubberdokMembers: {
+  [year: string]: {
+    name: string;
+    class: number;
+    position?: string;
+  }[];
+} = {
+  "2021/2022": [
+    { name: "Lars Waage", class: 4, position: "Prosjektleder" },
+    { name: "Hermann Mørkrid", class: 4, position: "Prosjektleder" },
+    { name: "Morgan Heggland", class: 5 },
+    { name: "Jorunn Leithe", class: 4 },
+    { name: "Jørgen Rosager", class: 4 },
+    { name: "Mathilde Marie Solberg", class: 4 },
+    { name: "Erik Thinn Tvedt", class: 3 },
+    { name: "Sebastian Cheng", class: 3 },
+    { name: "Herman Holmøy", class: 2 },
+    { name: "Ragnhild Bodsberg", class: 2 },
+    { name: "Johan Haga Mohn", class: 1 },
+  ],
+  "2020/2021": [
+    { name: "Morgan Heggland", class: 4, position: "Prosjektleder" },
+    { name: "Lars Lien Ankile", class: 4, position: "Lederpar HS" },
+    { name: "Andreas Johannesen", class: 4, position: "Lederpar HS" },
+    { name: "Sverre Spetalen", class: 5 },
+    { name: "Anna Sofie Lunde", class: 4 },
+    { name: "Fredrik Ahlborg", class: 4 },
+    { name: "Ingrid Aaseng", class: 4 },
+    { name: "Patrik Kjærran", class: 4 },
+    { name: "Hermann Mørkrid", class: 3 },
+    { name: "Lars Waage", class: 3 },
+    { name: "Mathilde Marie Solberg", class: 3 },
+    { name: "Herman Holmøy", class: 1 },
+    { name: "Mathias Raa", class: 1 },
+  ],
+};
+
+const useStyles = makeStyles((theme) => ({
+  closeButton: {
+    position: "absolute",
+    right: theme.spacing(1),
+    top: theme.spacing(1),
+    color: theme.palette.grey[500],
+  },
+  table: {
+    minWidth: 800,
+  },
+}));
+
+const HallOfFame: React.VFC<{
+  open: boolean;
+  setOpen: React.Dispatch<React.SetStateAction<boolean>>;
+}> = ({ open, setOpen }) => {
```

Comment:
I strongly prefer that we refactor all these into

```suggestion
type Props = {
  open: boolean;
  setOpen: React.Dispatch<React.SetStateAction<boolean>>;
}

const HallOfFame: React.VFC<Props> = ({ open, setOpen }) => {
```

To avoid having multiline arrow functions all over the place :)

Reply:
Agreed. Changed in 1085bf9. A bigger refactor should be done on all branches to ensure the use of separate Props type.